src/map/status.hpp

```diff
@@ -1257,6 +1257,21 @@ enum sc_type : int16 {
 
 	SC_WEAPONBREAKER,
 
+	SC_ALL_STAT_DOWN,
+	SC_GRADUAL_GRAVITY,
+	SC_DAMAGE_HEAL,
+	SC_IMMUNE_PROPERTY,
+	SC_IMMUNE_PROPERTY_NOTHING,
+	SC_IMMUNE_PROPERTY_WATER,
+	SC_IMMUNE_PROPERTY_GROUND,
+	SC_IMMUNE_PROPERTY_FIRE,
+	SC_IMMUNE_PROPERTY_WIND,
+	SC_IMMUNE_PROPERTY_POISON,
+	SC_IMMUNE_PROPERTY_SAINT,
+	SC_IMMUNE_PROPERTY_DARKNESS,
+	SC_IMMUNE_PROPERTY_TELEKINESIS,
+	SC_IMMUNE_PROPERTY_UNDEAD,
```

Comment:
I am a little 50:50 on if we should define 1 SC per element, as they cannot stack anyway.

Reply:
Yes, as I explained on top of this pull as todos, this skill is not completed. This skill's max level is 5, and level only affects skill's duration. You can check this in ```skillinfolist.lub``` of KRo.